### PR TITLE
Add react resources

### LINKED ISSUE #1374: [Add] <write what you want to add>
### Site URL

https://www.youtube.com/watch?v=v02P0AsTm2o&list=PLt5mNkGuWcuWSUHxSzWP74IU9U4BTVLt0&ab_channel=6PackProgrammer

### Description of the above provided site

Add more resources for react js by 6 pack programmer

### Category

Frontend

### Sub Category

designing

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by karthiknadar1204**
## Fixes Issue #
Closes #1374 

## Changes proposed

Added resources for react

### CODE REVIEW

File: database/frontend/react.json [hunk 1]
```diff
@@ -99,5 +99,13 @@
     "category": "frontend",
     "subcategory": "react",
     "language": "hindi"
+  },
+  {
+    "name": "React js",
+    "description": "In this Hindi video series, you will basics of the framework while creating projects on the go for better understanding of concepts and their working.",
```
Comment (rupali-codes) [inline database/frontend/react.json:105]:
```suggestion
    "description": "In this series, you will learn basics of the React.js while creating projects on the go for better understanding of concepts and their working.",

```

**Comment by rupali-codes** [comment]
@karthiknadar1204 please add _closes #issue_number_ in the description of this PR

**Comment by karthiknadar1204** [comment]
#1374

**Comment by karthiknadar1204** [comment]
@rupali-codes

**Comment by karthiknadar1204** [comment]
will push the changes again in a bit.
Thank you

**Comment by karthiknadar1204** [comment]
@rupali-codes Have pushed the changes

File: database/frontend/react.json [hunk 2]
```diff
@@ -99,5 +99,13 @@
     "category": "frontend",
     "subcategory": "react",
     "language": "hindi"
+  },
+  {
+    "name": "React js",
+    "description": "In this series, you will basics of React.js while creating projects on the go for better understanding of concepts and their working.",
```
Comment (CBID2) [inline database/frontend/react.json:105]:
```suggestion
    "description": "In this series, you will learn the basics of React.js while creating projects to gain a better understanding of concepts and their working.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]

**Review by CBID2 — APPROVED** [review]
Hi @karthiknadar1204! I made a few tweaks to your description, so it looks good now! :)

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**